odoo/tools/misc.py

```diff
@@ -1372,36 +1373,42 @@ def babel_locale_parse(lang_code):
         except:
             return babel.Locale.parse("en_US")
 
-def formatLang(env, value, digits=None, grouping=True, monetary=False, dp=False, currency_obj=False):
+def formatLang(env, value, digits=2, grouping=True, monetary=False, dp=None, currency_obj=None, rounding_mode=ROUND_HALF_EVEN, amount_rounding='decimals'):
```

Comment:
This is definitely a good opportunity to rewrite/improve the docstring :slightly_smiling_face: 

Also, I think `rounding_unit` is a bit more explicit than `amount_rounding`, what do you think?